apps/public-docsite-v9/src/Concepts/Theming.stories.mdx

```diff
@@ -145,3 +145,34 @@ import { themeToTokensObject } from '@fluentui/react-components';
 
 export const customTokens = themeToTokensObject(customLightTheme);
 ```
+
+### Creating theme styles without `FluentProvider`
+
+`createCSSRuleFromTheme()` allows you to create CSS from a theme and apply this CSS, for example, to `<body>`.
+
+It's framework agnostic and can be used without React. Safe for SSR scenarios:
```

Comment:
```suggestion
It's framework agnostic and can be used without React.
```

Please either remove this or update the example as it uses `window` which does not exist in Node env.

Reply:
Done! In general the function is safe for SSR though? Should we have an SSR example of does that to too technology-specific (i.e., you need a Next example, a Vite example, and so on)?